modules/core/client/config/core.client.routes.js

```diff
@@ -48,6 +48,10 @@ angular.module('core').config(['$stateProvider', '$urlRouterProvider',
             .state('subject-setup', {
                 url: '/subject-setup',
                 templateUrl: 'modules/core/views/subject_setup.client.view.html'
+            })
+            .state('reports', {
```

Comment:
!fix Please remember to pull #32 and get setup this route through the sidebar, rename the 'Records' button to 'Reports'